src/screens/NFTOffersSheet/OfferRow.tsx

```diff
@@ -145,18 +154,21 @@ export const OfferRow = ({ offer }: { offer: NftOffer }) => {
         });
         navigate(Routes.NFT_SINGLE_OFFER_SHEET, { offer });
       }}
+      style={{ marginVertical: 10, marginHorizontal: 20 }}
     >
-      <Columns space="16px" alignVertical="center">
-        <Column width="content">
-          <Box
+      <View
```

Comment:
RDS (except for `Text`)was removed from the `OfferRow` component to increase perf